src/components/SearchFilter.vue

```diff
@@ -30,18 +30,15 @@
           <b-form-checkbox :id="ids.bbox" v-model="provideBBox" @change="setBBox()">{{ $t('search.filterBySpatialExtent') }}</b-form-checkbox>
           <template>
             <b-form-group v-if="provideBBox">
-              <b-form-radio-group
-                v-model="bboxSelectionStyle"
-                buttons
-                size="sm"
-                button-variant="outline-primary"
-              >
-                <b-form-radio value="map">{{ $t('search.defineBbox.map') }}</b-form-radio>
-                <b-form-radio value="text">{{ $t('search.defineBbox.text') }}</b-form-radio>
-              </b-form-radio-group>
+              <b-tabs>
+                <b-tab v-model="bboxSelectionStyle" :title="$t('search.defineBbox.map')">
+                  <Map class="mb-4" :stac="stac" selectBounds @bounds="setBBox" scrollWheelZoom />
+                </b-tab>
+                <b-tab v-model="bboxSelectionStyle" :title="$t('search.defineBbox.text')">
+                  <BBoxEntry :bbox="query.bbox || [-180, -80, 180, 80]" @updateBBoxArray="updateBBoxArray" />
```

Comment:
Defaults should be specified in the component prop, not in the template.

Reply:
done.  Because the `query.bbox` is null by default I now let the component handle null values. I also added validation that if passed an array, the array must be an array of 4 numbers.